src/current/molt/molt-fetch.md

```diff
@@ -657,37 +760,6 @@ molt fetch tokens list \
 Continuation Tokens.
```

Comment:
Is that `Continuation Tokens.` part of the output? It just looks strange under the table.

Reply:
Yep!